client/src/components/Layout.test.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import Footer from './Footer';
+import Header from './Header';
+import Layout from './Layout';
+import { Helmet } from "react-helmet";
+import { Toaster } from 'react-hot-toast';
+import { render, screen } from "@testing-library/react";
+import '@testing-library/jest-dom/extend-expect';
+
+jest.mock('react-hot-toast', () => {
+    return {
+        Toaster: () => <div>Toaster Mock</div> 
+    };
+});
+
+jest.mock('react-helmet', () => {
+    return {
+        Helmet: ({ children }) => 
+            <div>{children}</div>,
+    };
+});
+
+jest.mock('./Footer', () => () => <div>Footer Mock</div>);
+jest.mock('./Header', () => () => <div>Header Mock</div>);
+
+describe('Layout component', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    })
+
+    test('should render Helmet component properly', () => {
+        render(
+            <Layout
+                title='Test Title'
+                description='Test Description'
+                keywords='Test Keywords'
+                author='Test Author'
+            >
+                <div>Child</div>
+            </Layout>
+        )
+
+        expect(screen.getByText('Test Title')).toBeInTheDocument();
+        expect(document.querySelector('meta[name="description"]').getAttribute('content')).toBe('Test Description');
+        expect(document.querySelector('meta[name="keywords"]').getAttribute('content')).toBe('Test Keywords');
+        expect(document.querySelector('meta[name="author"]').getAttribute('content')).toBe('Test Author');
+    })
+
+    test('should render Header, Footer, Toaster and Children correctly', () => {
```

Comment:
For this, since you are rendering `<Layout>` without any props, by default it should use the default props. You might want to assert whether the `content` in the `meta` tags are set correctly in such cases, because in your previous test case, you explicitly passed in the prop values, which does not rely on the default ones.

Reply:
Okie, I have added a default test case!